aws-rds/README.md

```diff
@@ -24,81 +24,181 @@ Use SignalFx to monitor Amazon Relational Database Service (RDS) via [Amazon Web
-
-There are two ways to deploy the Lambda function:
-
-- Option 1: Deploy from the Serverless Application Repository
-    - SignalFx recommends this option.
-
-- Option 2: Deploy (building) from the source
-
-To learn how to deploy the Lambda function for Enhanced Monitoring, see <a target="_blank" href="https://github.com/signalfx/enhanced-rds-monitoring">SignalFx Enhanced RDS Monitoring Integration</a>.
+### 4. Deploy the function and configure the trigger
+Click `Deploy`. After the function has finished deploying, navigate to the
+function's main page.
+
+Under the `Configuration` tab, scroll through the list on the left and
+select CloudWatch Logs as the source of the trigger. You can then configure the trigger:
+
+* Select `RDSOSMetrics` as the log group.
+* Choose an appropriate name for the filter, and leave the filter pattern
+blank.
+* Make sure the `Enabled` switch is activated.
+
+Click `Add`, then click `Save` in the upper right corner.
+
+That's it! Your metrics are on the way to SignalFx ingest!
+
+## Building from source
+
+### 1. Set up the execution role
+The execution role just needs basic Lambda execution permissions and KMS
+decrypt permissions (if you wish to encrypt your SignalFx access token). If you
+don't want to create one, you can select from a list of templates when you
+create the lambda function.
+
+### 2. Set up an encryption key and encrypt access token
+Only follow this step if you chose to encrypt your access token. Either create
+a new KMS encryption key or select a preexisting one. **The key must be in the
+same availability zone as the RDS instances you are monitoring.** You can
+create and manage encryption keys from IAM in the AWS management console.
+Documentation on KMS encryption from the CLI can be found
+[here](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html).
+Make sure you have access to the cipher text output by the encryption as well
+as the key id of the encryption key you used.
+
+### 3. Clone the source repo and build the deployment package
+You can find the repo
+[here](https://github.com/signalfx/enhanced-rds-monitoring).
+After you have cloned the repo, do the following:
+```
+$ cd enhanced-rds-monitoring
+$ ./build.sh
+```
+The package will be named `enhanced_rds.zip`. This is the file to upload for the Lambda.
+
+### 4. Create and configure the Lambda function
+From the Lambda creation screen, make sure you have selected
+`Build from scratch`. Select a name for your function. For `Runtime` select
+`Python3.8` (although `Python3.6` and `Python3.7` are also supported). For the execution role, either select the role you want to use or
+select `Create from Template` and add KMS decrypt permissions if necessary. You
+will also need to choose a name for the role.
+
+For subsequent tabs, follow the instructions below.
+
+#### Designer
+The only thing to be done here is to set up the trigger from CloudWatch Logs:
```

Comment:
Delete "The only thing to be done here." I would start with "To set up the trigger..."